diff --git a/src/app/qgscustomization.cpp b/src/app/qgscustomization.cpp
index faaed2b..f25840d 100644
--- a/src/app/qgscustomization.cpp
+++ b/src/app/qgscustomization.cpp
@@ -652,6 +652,25 @@ void QgsCustomization::updateMainWindow( QMenu * theToolBarMenu )
   QMainWindow* mw = QgisApp::instance();
   QMenuBar* menubar = mw->menuBar();
 
+  // hide Panels
+  mSettings->beginGroup( "Customization/Panels" );
+  foreach ( QObject* obj, mw->children() )
+  {
+    if ( obj->inherits( "QDockWidget" ) )
+    {
+	  QDockWidget* bdw = qobject_cast<QDockWidget*>( obj );
+	  bool visible = mSettings->value( bdw->objectName(), true ).toBool();
+	
+      if ( !visible )
+      {
+		  mw->removeDockWidget(bdw);
+		  QgisApp::instance()->panelMenu()->removeAction( bdw->toggleViewAction() );
+      }
+    }
+  }
+
+  mSettings->endGroup();
+
   mSettings->beginGroup( "Customization/Menus" );
 
   // hide menus and menu actions
