diff --git a/i18n/qgis_ru.ts b/i18n/qgis_ru.ts
index efe34d6..d5ab5f2 100644
--- a/i18n/qgis_ru.ts
+++ b/i18n/qgis_ru.ts
@@ -76651,4 +76651,26 @@ Convert a string into Qt data type.
         <translation>...</translation>
     </message>
 </context>
+<context>
+    <name>QgisApp</name>
+    <message>
+        <source>Target monitoring system of regional ecosystem</source>
+        <translation>Целевая система мониторинга региональной экосистемы</translation>
+    </message>
+</context>
+<context>
+    <name>QgsAbout</name>
+    <message>
+        <source>RIAS RKD</source>
+        <translation>РИАС РКД</translation>
+    </message>
+    <message>
+        <source>TMSforRES</source>
+        <translation>ЦСМ РЭС</translation>
+    </message>
+    <message>
+        <source>Core description</source>
+        <translation>Описание ядра</translation>
+    </message>
+</context>
 </TS>
diff --git a/images/icons/qgis-icon-16x16.png b/images/icons/qgis-icon-16x16.png
index 2fe7f2e..da1b058 100644
Binary files a/images/icons/qgis-icon-16x16.png and b/images/icons/qgis-icon-16x16.png differ
diff --git a/images/icons/qgis-icon-60x60.png b/images/icons/qgis-icon-60x60.png
index a21665c..e91d8db 100644
Binary files a/images/icons/qgis-icon-60x60.png and b/images/icons/qgis-icon-60x60.png differ
diff --git a/images/images.qrc b/images/images.qrc
index 39e3172..2f55688 100644
--- a/images/images.qrc
+++ b/images/images.qrc
@@ -473,4 +473,7 @@
         <file>sponsors/platinum.png</file>
         <file>sponsors/silver.png</file>
     </qresource>
+    <qresource prefix="/images/rekod_tms_4_doc">
+        <file alias="TMSforRES.jpg">rekod_tms_4_doc/TMSforRES.jpg</file>
+    </qresource>
 </RCC>
diff --git a/images/splash/splash.png b/images/splash/splash.png
index 54b9c07..3ab834d 100644
Binary files a/images/splash/splash.png and b/images/splash/splash.png differ
diff --git a/images/themes/default/mActionHelpAbout.png b/images/themes/default/mActionHelpAbout.png
index 27ff9d3..da1b058 100644
Binary files a/images/themes/default/mActionHelpAbout.png and b/images/themes/default/mActionHelpAbout.png differ
diff --git a/src/app/main.cpp b/src/app/main.cpp
index 3220506..12b2965 100644
--- a/src/app/main.cpp
+++ b/src/app/main.cpp
@@ -478,6 +478,7 @@ int main( int argc, char *argv[] )
   QString pythonfile;
 
   QString customizationfile;
+  QString myWorkspace;
 
 #if defined(ANDROID)
   QgsDebugMsg( QString( "Android: All params stripped" ) );// Param %1" ).arg( argv[0] ) );
@@ -560,6 +561,10 @@ int main( int argc, char *argv[] )
       {
         myRestoreDefaultWindowState = true;
       }
+	  else if ( arg == "--workspace")
+      {
+        myWorkspace = QDir::toNativeSeparators( QDir( args[++i] ).absolutePath() );;
+      }
       else
       {
         myFileList.append( QDir::toNativeSeparators( QFileInfo( args[i] ).absoluteFilePath() ) );
@@ -688,6 +693,8 @@ int main( int argc, char *argv[] )
     }
   }
 
+  // set workspace
+  mySettings.setValue("/UI/lastProjectDir", myWorkspace);
 
   // custom environment variables
   QMap<QString, QString> systemEnvVars = QgsApplication::systemEnvVars();
diff --git a/src/app/qgis.ico b/src/app/qgis.ico
index 87d76d1..60cbad7 100644
Binary files a/src/app/qgis.ico and b/src/app/qgis.ico differ
diff --git a/src/app/qgisapp.cpp b/src/app/qgisapp.cpp
index a72bba7..946bafa 100644
--- a/src/app/qgisapp.cpp
+++ b/src/app/qgisapp.cpp
@@ -340,6 +340,7 @@ const int AFTER_RECENT_PATHS = 321;
   */
 static void setTitleBarText_( QWidget & qgisApp )
 {
+  /*
   QString caption = QgisApp::tr( "QGIS " );
 
   if ( QString( QGis::QGIS_VERSION ).endsWith( "Master" ) )
@@ -368,7 +369,8 @@ static void setTitleBarText_( QWidget & qgisApp )
   {
     caption += " - " + QgsProject::instance()->title();
   }
-
+  */
+  QString caption = QgisApp::tr( "Target monitoring system of regional ecosystem" );
   qgisApp.setWindowTitle( caption );
 } // setTitleBarText_( QWidget * qgisApp )
 
@@ -703,7 +705,10 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, QWidget * parent,
   connect( QgsMapLayerActionRegistry::instance(), SIGNAL( changed() ), this, SLOT( refreshActionFeatureAction() ) );
 
   // set application's caption
+  /*
   QString caption = tr( "QGIS - %1 ('%2')" ).arg( QGis::QGIS_VERSION ).arg( QGis::QGIS_RELEASE_NAME );
+  */
+  QString caption = QgisApp::tr( "Target monitoring system of regional ecosystem" );
   setWindowTitle( caption );
 
   QgsMessageLog::logMessage( tr( "QGIS starting..." ), QString::null, QgsMessageLog::INFO );
diff --git a/src/app/qgsabout.cpp b/src/app/qgsabout.cpp
index 1ff924a..4ae6723 100644
--- a/src/app/qgsabout.cpp
+++ b/src/app/qgsabout.cpp
@@ -1,305 +1,356 @@
-/***************************************************************************
-                          qgsabout.cpp  -  description
-                             -------------------
-    begin                : Sat Aug 10 2002
-    copyright            : (C) 2002 by Gary E.Sherman
-    email                : sherman at mrcc.com
- ***************************************************************************/
-
-/***************************************************************************
- *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-
-#include "qgsabout.h"
-#include "qgsapplication.h"
-#include "qgsproviderregistry.h"
-#include "qgslogger.h"
-#include <QDesktopServices>
-#include <QFile>
-#include <QTextStream>
-#include <QImageReader>
-#include <QSqlDatabase>
-
-/* Uncomment this block to use preloaded images
-#include <map>
-std::map<QString, QPixmap> mugs;
-*/
-#ifdef Q_OS_MACX
-QgsAbout::QgsAbout( QWidget *parent )
-    : QgsOptionsDialogBase( "about", parent, Qt::WindowSystemMenuHint )  // Modeless dialog with close button only
-#else
-QgsAbout::QgsAbout( QWidget *parent )
-    : QgsOptionsDialogBase( "about", parent )  // Normal dialog in non Mac-OS
-#endif
-{
-  setupUi( this );
-  QString title = QString( "%1 - %2 Bit" ).arg( windowTitle() ).arg( QSysInfo::WordSize );
-  initOptionsBase( true, title );
-  init();
-}
-
-QgsAbout::~QgsAbout()
-{
-}
-
-void QgsAbout::init()
-{
-  setPluginInfo();
-
-  // set the 60x60 icon pixmap
-  QPixmap icon( QgsApplication::iconsPath() + "qgis-icon-60x60.png" );
-  qgisIcon->setPixmap( icon );
-
-  //read the authors file to populate the svn committers list
-  QStringList lines;
-
-  //
-  // Load the authors (svn committers) list
-  //
-  QFile file( QgsApplication::authorsFilePath() );
-  if ( file.open( QIODevice::ReadOnly ) )
-  {
-    QTextStream stream( &file );
-    // Always use UTF-8
-    stream.setCodec( "UTF-8" );
-    QString line;
-    while ( !stream.atEnd() )
-    {
-      line = stream.readLine(); // line of text excluding '\n'
-      //ignore the line if it starts with a hash....
-      if ( line.left( 1 ) == "#" )
-        continue;
-      QStringList myTokens = line.split( "\t", QString::SkipEmptyParts );
-      lines << myTokens[0];
-    }
-    file.close();
-    lstDevelopers->clear();
-    lstDevelopers->insertItems( 0, lines );
-
-    if ( lstDevelopers->count() > 0 )
-    {
-      lstDevelopers->setCurrentRow( 0 );
-    }
-  }
-
-  lines.clear();
-  //
-  // Now load up the contributors list
-  //
-  QFile file2( QgsApplication::contributorsFilePath() );
-  printf( "Reading contributors file %s.............................................\n",
-          file2.fileName().toLocal8Bit().constData() );
-  if ( file2.open( QIODevice::ReadOnly ) )
-  {
-    QTextStream stream( &file2 );
-    // Always use UTF-8
-    stream.setCodec( "UTF-8" );
-    QString line;
-    while ( !stream.atEnd() )
-    {
-      line = stream.readLine(); // line of text excluding '\n'
-      //ignore the line if it starts with a hash....
-      if ( line.left( 1 ) == "#" )
-        continue;
-      lines += line;
-    }
-    file2.close();
-    lstContributors->clear();
-    lstContributors->insertItems( 0, lines );
-    if ( lstContributors->count() > 0 )
-    {
-      lstContributors->setCurrentRow( 0 );
-    }
-  }
-
-
-
-  // read the DONORS file and populate the text widget
-  QFile donorsFile( QgsApplication::donorsFilePath() );
-#ifdef QGISDEBUG
-  printf( "Reading donors file %s.............................................\n",
-          donorsFile.fileName().toLocal8Bit().constData() );
-#endif
-  if ( donorsFile.open( QIODevice::ReadOnly ) )
-  {
-    QString donorsHTML = ""
-                         + tr( "<p>For a list of individuals and institutions who have contributed "
-                               "money to fund QGIS development and other project costs see "
-                               "<a href=\"http://qgis.org/en/site/about/sponsorship.html#list-of-donors\">"
-                               "http://qgis.org/en/site/about/sponsorship.html#list-of-donors</a></p>" );
-#if 0
-    QString website;
-    QTextStream donorsStream( &donorsFile );
-    // Always use UTF-8
-    donorsStream.setCodec( "UTF-8" );
-    QString sline;
-    while ( !donorsStream.atEnd() )
-    {
-      sline = donorsStream.readLine(); // line of text excluding '\n'
-      //ignore the line if it starts with a hash....
-      if ( sline.left( 1 ) == "#" )
-        continue;
-      QStringList myTokens = sline.split( "|", QString::SkipEmptyParts );
-      if ( myTokens.size() > 1 )
-      {
-        website = "<a href=\"" + myTokens[1].remove( ' ' ) + "\">" + myTokens[1] + "</a>";
-      }
-      else
-      {
-        website = "&nbsp;";
-      }
-      donorsHTML += "<tr>";
-      donorsHTML += "<td>" + myTokens[0] + "</td><td>" + website + "</td>";
-      // close the row
-      donorsHTML += "</tr>";
-    }
-    donorsHTML += "</table>";
-#endif
-
-    QString myStyle = QgsApplication::reportStyleSheet();
-    txtDonors->clear();
-    txtDonors->document()->setDefaultStyleSheet( myStyle );
-    txtDonors->setHtml( donorsHTML );
-    QgsDebugMsg( QString( "donorsHTML:%1" ).arg( donorsHTML.toAscii().constData() ) );
-  }
-
-  // read the TRANSLATORS file and populate the text widget
-  QFile translatorFile( QgsApplication::translatorsFilePath() );
-#ifdef QGISDEBUG
-  printf( "Reading translators file %s.............................................\n",
-          translatorFile.fileName().toLocal8Bit().constData() );
-#endif
-  if ( translatorFile.open( QIODevice::ReadOnly ) )
-  {
-    QString translatorHTML = "";
-    QTextStream translatorStream( &translatorFile );
-    // Always use UTF-8
-    translatorStream.setCodec( "UTF-8" );
-    QString myStyle = QgsApplication::reportStyleSheet();
-    translatorHTML += "<style>" + myStyle + "</style>";
-    while ( !translatorStream.atEnd() )
-    {
-      translatorHTML += translatorStream.readLine();
-    }
-    txtTranslators->setHtml( translatorHTML );
-    QgsDebugMsg( QString( "translatorHTML:%1" ).arg( translatorHTML.toAscii().constData() ) );
-  }
-  setWhatsNew();
-  setLicence();
-}
-
-void QgsAbout::setLicence()
-{
-  // read the DONORS file and populate the text widget
-  QFile licenceFile( QgsApplication::licenceFilePath() );
-#ifdef QGISDEBUG
-  printf( "Reading licence file %s.............................................\n",
-          licenceFile.fileName().toLocal8Bit().constData() );
-#endif
-  if ( licenceFile.open( QIODevice::ReadOnly ) )
-  {
-    QString content = licenceFile.readAll();
-    txtLicense->setText( content );
-  }
-}
-
-void QgsAbout::setVersion( QString v )
-{
-  txtVersion->setBackgroundRole( QPalette::NoRole );
-  txtVersion->setAutoFillBackground( true );
-  txtVersion->setHtml( v );
-}
-
-void QgsAbout::setWhatsNew( )
-{
-  QString myStyle = QgsApplication::reportStyleSheet();
-  txtWhatsNew->clear();
-  txtWhatsNew->document()->setDefaultStyleSheet( myStyle );
-  txtWhatsNew->setSource( "file:///" + QgsApplication::pkgDataPath() + "/doc/news.html" );
-}
-
-void QgsAbout::setPluginInfo()
-{
-  QString myString;
-  //provide info about the plugins available
-  myString += "<b>" + tr( "Available QGIS Data Provider Plugins" ) + "</b><br>";
-  myString += QgsProviderRegistry::instance()->pluginList( true );
-  //qt database plugins
-  myString += "<b>" + tr( "Available Qt Database Plugins" ) + "</b><br>";
-  myString += "<ol>\n<li>\n";
-  QStringList myDbDriverList = QSqlDatabase::drivers();
-  myString += myDbDriverList.join( "</li>\n<li>" );
-  myString += "</li>\n</ol>\n";
-  //qt image plugins
-  myString += "<b>" + tr( "Available Qt Image Plugins" ) + "</b><br>";
-  myString += tr( "Qt Image Plugin Search Paths <br>" );
-  myString += QApplication::libraryPaths().join( "<br>" );
-  myString += "<ol>\n<li>\n";
-  QList<QByteArray> myImageFormats = QImageReader::supportedImageFormats();
-  QList<QByteArray>::const_iterator myIterator = myImageFormats.begin();
-  while ( myIterator != myImageFormats.end() )
-  {
-    QString myFormat = ( *myIterator ).data();
-    myString += myFormat + "</li>\n<li>";
-    ++myIterator;
-  }
-  myString += "</li>\n</ol>\n";
-
-  QString myStyle = QgsApplication::reportStyleSheet();
-  txtProviders->clear();
-  txtProviders->document()->setDefaultStyleSheet( myStyle );
-  txtProviders->setText( myString );
-}
-
-void QgsAbout::on_btnQgisUser_clicked()
-{
-  // find a browser
-  QString url = "http://lists.osgeo.org/mailman/listinfo/qgis-user";
-  openUrl( url );
-}
-
-void QgsAbout::on_btnQgisHome_clicked()
-{
-  openUrl( "http://qgis.org" );
-}
-
-void QgsAbout::openUrl( QString url )
-{
-  //use the users default browser
-  QDesktopServices::openUrl( url );
-}
-
-/*
- * The function below makes a name safe for using in most file system
- * Step 1: Code QString as UTF-8
- * Step 2: Replace all bytes of the UTF-8 above 0x7f with the hexcode in lower case.
- * Step 2: Replace all non [a-z][a-Z][0-9] with underscore (backward compatibility)
- */
-QString QgsAbout::fileSystemSafe( QString fileName )
-{
-  QString result;
-  QByteArray utf8 = fileName.toUtf8();
-
-  for ( int i = 0; i < utf8.size(); i++ )
-  {
-    uchar c = utf8[i];
-
-    if ( c > 0x7f )
-    {
-      result = result + QString( "%1" ).arg( c, 2, 16, QChar( '0' ) );
-    }
-    else
-    {
-      result = result + QString( c );
-    }
-  }
-  result.replace( QRegExp( "[^a-z0-9A-Z]" ), "_" );
-  QgsDebugMsg( result );
-
-  return result;
-}
+/***************************************************************************
+                          qgsabout.cpp  -  description
+                             -------------------
+    begin                : Sat Aug 10 2002
+    copyright            : (C) 2002 by Gary E.Sherman
+    email                : sherman at mrcc.com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsabout.h"
+#include "qgsapplication.h"
+#include "qgsproviderregistry.h"
+#include "qgslogger.h"
+#include <QDesktopServices>
+#include <QFile>
+#include <QTextStream>
+#include <QImageReader>
+#include <QSqlDatabase>
+
+/* Uncomment this block to use preloaded images
+#include <map>
+std::map<QString, QPixmap> mugs;
+*/
+#ifdef Q_OS_MACX
+QgsAbout::QgsAbout( QWidget *parent )
+    : QgsOptionsDialogBase( "about", parent, Qt::WindowSystemMenuHint )  // Modeless dialog with close button only
+#else
+QgsAbout::QgsAbout( QWidget *parent )
+    : QgsOptionsDialogBase( "about", parent )  // Normal dialog in non Mac-OS
+#endif
+{
+  setupUi( this );
+
+  QSettings qgisSettings;
+
+  TextLabel4->setText(TextLabel4->text() + " (build for Rekod, version 0.0.8) ");
+
+  // RAIS RKD about -------------------------
+  const QString RAIS_RKD_About_FileName(QgsApplication::recodAboutFilesPath() + "/RIAS_RKD_About");
+
+  QFile RIASRKDFile( RAIS_RKD_About_FileName );
+
+  QString riasrkdHTML = "";
+  if ( RIASRKDFile.open( QIODevice::ReadOnly ) )
+  {
+	
+    QTextStream translatorStream( &RIASRKDFile );
+    translatorStream.setCodec( "UTF-8" );
+    while ( !translatorStream.atEnd() )
+    {
+      riasrkdHTML += translatorStream.readLine();
+    }
+  }
+  else
+  {
+	riasrkdHTML += "<h1>" + RAIS_RKD_About_FileName + " not found </h1>";
+  }
+  RAIS_RKD_About_tb->setHtml( riasrkdHTML );
+  // -----------------------------------
+
+  // TMS about -------------------------  
+  const QString TMS_About_FileName(QgsApplication::recodAboutFilesPath() + "/TMSforRES_About");
+  
+  QFile TMSFile( TMS_About_FileName );
+
+  QString TMSHTML = "";
+  if ( TMSFile.open( QIODevice::ReadOnly ) )
+  {
+	
+    QTextStream translatorStream( &TMSFile );
+    translatorStream.setCodec( "UTF-8" );
+    while ( !translatorStream.atEnd() )
+    {
+      TMSHTML += translatorStream.readLine();
+    }
+  }
+  else
+  {
+	TMSHTML += "<h1>" + TMS_About_FileName + " not found </h1>";
+  }
+  TMS_About_tb->setHtml( TMSHTML );
+  // -----------------------------------
+
+  QString title = QString( "%1 - %2 Bit" ).arg( windowTitle() ).arg( QSysInfo::WordSize );
+  initOptionsBase( true, title );
+  init();
+}
+
+QgsAbout::~QgsAbout()
+{
+}
+
+void QgsAbout::init()
+{
+  setPluginInfo();
+
+  // set the 60x60 icon pixmap
+  QPixmap icon( QgsApplication::iconsPath() + "qgis-icon-60x60.png" );
+  qgisIcon->setPixmap( icon );
+
+  //read the authors file to populate the svn committers list
+  QStringList lines;
+
+  //
+  // Load the authors (svn committers) list
+  //
+  QFile file( QgsApplication::authorsFilePath() );
+  if ( file.open( QIODevice::ReadOnly ) )
+  {
+    QTextStream stream( &file );
+    // Always use UTF-8
+    stream.setCodec( "UTF-8" );
+    QString line;
+    while ( !stream.atEnd() )
+    {
+      line = stream.readLine(); // line of text excluding '\n'
+      //ignore the line if it starts with a hash....
+      if ( line.left( 1 ) == "#" )
+        continue;
+      QStringList myTokens = line.split( "\t", QString::SkipEmptyParts );
+      lines << myTokens[0];
+    }
+    file.close();
+    lstDevelopers->clear();
+    lstDevelopers->insertItems( 0, lines );
+
+    if ( lstDevelopers->count() > 0 )
+    {
+      lstDevelopers->setCurrentRow( 0 );
+    }
+  }
+
+  lines.clear();
+  //
+  // Now load up the contributors list
+  //
+  QFile file2( QgsApplication::contributorsFilePath() );
+  printf( "Reading contributors file %s.............................................\n",
+          file2.fileName().toLocal8Bit().constData() );
+  if ( file2.open( QIODevice::ReadOnly ) )
+  {
+    QTextStream stream( &file2 );
+    // Always use UTF-8
+    stream.setCodec( "UTF-8" );
+    QString line;
+    while ( !stream.atEnd() )
+    {
+      line = stream.readLine(); // line of text excluding '\n'
+      //ignore the line if it starts with a hash....
+      if ( line.left( 1 ) == "#" )
+        continue;
+      lines += line;
+    }
+    file2.close();
+    lstContributors->clear();
+    lstContributors->insertItems( 0, lines );
+    if ( lstContributors->count() > 0 )
+    {
+      lstContributors->setCurrentRow( 0 );
+    }
+  }
+
+
+
+  // read the DONORS file and populate the text widget
+  QFile donorsFile( QgsApplication::donorsFilePath() );
+#ifdef QGISDEBUG
+  printf( "Reading donors file %s.............................................\n",
+          donorsFile.fileName().toLocal8Bit().constData() );
+#endif
+  if ( donorsFile.open( QIODevice::ReadOnly ) )
+  {
+    QString donorsHTML = ""
+                         + tr( "<p>For a list of individuals and institutions who have contributed "
+                               "money to fund QGIS development and other project costs see "
+                               "<a href=\"http://qgis.org/en/site/about/sponsorship.html#list-of-donors\">"
+                               "http://qgis.org/en/site/about/sponsorship.html#list-of-donors</a></p>" );
+#if 0
+    QString website;
+    QTextStream donorsStream( &donorsFile );
+    // Always use UTF-8
+    donorsStream.setCodec( "UTF-8" );
+    QString sline;
+    while ( !donorsStream.atEnd() )
+    {
+      sline = donorsStream.readLine(); // line of text excluding '\n'
+      //ignore the line if it starts with a hash....
+      if ( sline.left( 1 ) == "#" )
+        continue;
+      QStringList myTokens = sline.split( "|", QString::SkipEmptyParts );
+      if ( myTokens.size() > 1 )
+      {
+        website = "<a href=\"" + myTokens[1].remove( ' ' ) + "\">" + myTokens[1] + "</a>";
+      }
+      else
+      {
+        website = "&nbsp;";
+      }
+      donorsHTML += "<tr>";
+      donorsHTML += "<td>" + myTokens[0] + "</td><td>" + website + "</td>";
+      // close the row
+      donorsHTML += "</tr>";
+    }
+    donorsHTML += "</table>";
+#endif
+
+    QString myStyle = QgsApplication::reportStyleSheet();
+    txtDonors->clear();
+    txtDonors->document()->setDefaultStyleSheet( myStyle );
+    txtDonors->setHtml( donorsHTML );
+    QgsDebugMsg( QString( "donorsHTML:%1" ).arg( donorsHTML.toAscii().constData() ) );
+  }
+
+  // read the TRANSLATORS file and populate the text widget
+  QFile translatorFile( QgsApplication::translatorsFilePath() );
+#ifdef QGISDEBUG
+  printf( "Reading translators file %s.............................................\n",
+          translatorFile.fileName().toLocal8Bit().constData() );
+#endif
+  if ( translatorFile.open( QIODevice::ReadOnly ) )
+  {
+    QString translatorHTML = "";
+    QTextStream translatorStream( &translatorFile );
+    // Always use UTF-8
+    translatorStream.setCodec( "UTF-8" );
+    QString myStyle = QgsApplication::reportStyleSheet();
+    translatorHTML += "<style>" + myStyle + "</style>";
+    while ( !translatorStream.atEnd() )
+    {
+      translatorHTML += translatorStream.readLine();
+    }
+    txtTranslators->setHtml( translatorHTML );
+    QgsDebugMsg( QString( "translatorHTML:%1" ).arg( translatorHTML.toAscii().constData() ) );
+  }
+  setWhatsNew();
+  setLicence();
+}
+
+void QgsAbout::setLicence()
+{
+  // read the DONORS file and populate the text widget
+  QFile licenceFile( QgsApplication::licenceFilePath() );
+#ifdef QGISDEBUG
+  printf( "Reading licence file %s.............................................\n",
+          licenceFile.fileName().toLocal8Bit().constData() );
+#endif
+  if ( licenceFile.open( QIODevice::ReadOnly ) )
+  {
+    QString content = licenceFile.readAll();
+    txtLicense->setText( content );
+  }
+}
+
+void QgsAbout::setVersion( QString v )
+{
+  txtVersion->setBackgroundRole( QPalette::NoRole );
+  txtVersion->setAutoFillBackground( true );
+  txtVersion->setHtml( v );
+}
+
+void QgsAbout::setWhatsNew( )
+{
+  QString myStyle = QgsApplication::reportStyleSheet();
+  txtWhatsNew->clear();
+  txtWhatsNew->document()->setDefaultStyleSheet( myStyle );
+  txtWhatsNew->setSource( "file:///" + QgsApplication::pkgDataPath() + "/doc/news.html" );
+}
+
+void QgsAbout::setPluginInfo()
+{
+  QString myString;
+  //provide info about the plugins available
+  myString += "<b>" + tr( "Available QGIS Data Provider Plugins" ) + "</b><br>";
+  myString += QgsProviderRegistry::instance()->pluginList( true );
+  //qt database plugins
+  myString += "<b>" + tr( "Available Qt Database Plugins" ) + "</b><br>";
+  myString += "<ol>\n<li>\n";
+  QStringList myDbDriverList = QSqlDatabase::drivers();
+  myString += myDbDriverList.join( "</li>\n<li>" );
+  myString += "</li>\n</ol>\n";
+  //qt image plugins
+  myString += "<b>" + tr( "Available Qt Image Plugins" ) + "</b><br>";
+  myString += tr( "Qt Image Plugin Search Paths <br>" );
+  myString += QApplication::libraryPaths().join( "<br>" );
+  myString += "<ol>\n<li>\n";
+  QList<QByteArray> myImageFormats = QImageReader::supportedImageFormats();
+  QList<QByteArray>::const_iterator myIterator = myImageFormats.begin();
+  while ( myIterator != myImageFormats.end() )
+  {
+    QString myFormat = ( *myIterator ).data();
+    myString += myFormat + "</li>\n<li>";
+    ++myIterator;
+  }
+  myString += "</li>\n</ol>\n";
+
+  QString myStyle = QgsApplication::reportStyleSheet();
+  txtProviders->clear();
+  txtProviders->document()->setDefaultStyleSheet( myStyle );
+  txtProviders->setText( myString );
+}
+
+void QgsAbout::on_btnQgisUser_clicked()
+{
+  // find a browser
+  QString url = "http://lists.osgeo.org/mailman/listinfo/qgis-user";
+  openUrl( url );
+}
+
+void QgsAbout::on_btnQgisHome_clicked()
+{
+  openUrl( "http://qgis.org" );
+}
+
+void QgsAbout::openUrl( QString url )
+{
+  //use the users default browser
+  QDesktopServices::openUrl( url );
+}
+
+/*
+ * The function below makes a name safe for using in most file system
+ * Step 1: Code QString as UTF-8
+ * Step 2: Replace all bytes of the UTF-8 above 0x7f with the hexcode in lower case.
+ * Step 2: Replace all non [a-z][a-Z][0-9] with underscore (backward compatibility)
+ */
+QString QgsAbout::fileSystemSafe( QString fileName )
+{
+  QString result;
+  QByteArray utf8 = fileName.toUtf8();
+
+  for ( int i = 0; i < utf8.size(); i++ )
+  {
+    uchar c = utf8[i];
+
+    if ( c > 0x7f )
+    {
+      result = result + QString( "%1" ).arg( c, 2, 16, QChar( '0' ) );
+    }
+    else
+    {
+      result = result + QString( c );
+    }
+  }
+  result.replace( QRegExp( "[^a-z0-9A-Z]" ), "_" );
+  QgsDebugMsg( result );
+
+  return result;
+}
diff --git a/src/app/qgscustomization.cpp b/src/app/qgscustomization.cpp
index 460d8e4..a78862a 100644
--- a/src/app/qgscustomization.cpp
+++ b/src/app/qgscustomization.cpp
@@ -652,6 +652,25 @@ void QgsCustomization::updateMainWindow( QMenu * theToolBarMenu )
   QMainWindow* mw = QgisApp::instance();
   QMenuBar* menubar = mw->menuBar();
 
+  // hide Panels
+  mSettings->beginGroup( "Customization/Panels" );
+  foreach ( QObject* obj, mw->children() )
+  {
+    if ( obj->inherits( "QDockWidget" ) )
+    {
+	  QDockWidget* bdw = qobject_cast<QDockWidget*>( obj );
+	  bool visible = mSettings->value( bdw->objectName(), true ).toBool();
+	
+      if ( !visible )
+      {
+		  mw->removeDockWidget(bdw);
+		  QgisApp::instance()->panelMenu()->removeAction( bdw->toggleViewAction() );
+      }
+    }
+  }
+
+  mSettings->endGroup();
+
   mSettings->beginGroup( "Customization/Menus" );
 
   // hide menus and menu actions
diff --git a/src/core/qgsapplication.cpp b/src/core/qgsapplication.cpp
index 4e928d3..6252df3 100644
--- a/src/core/qgsapplication.cpp
+++ b/src/core/qgsapplication.cpp
@@ -464,6 +464,14 @@ const QString QgsApplication::licenceFilePath()
 }
 
 /*!
+  Returns the path to the Recod doc files.
+*/
+const QString QgsApplication::recodAboutFilesPath()
+{
+  return ABISYM( mPkgDataPath ) + QString( "/doc/Rekod" );
+}
+
+/*!
   Returns the path to the help application.
 */
 const QString QgsApplication::helpAppPath()
diff --git a/src/core/qgsapplication.h b/src/core/qgsapplication.h
index c4b4676..41fd197 100644
--- a/src/core/qgsapplication.h
+++ b/src/core/qgsapplication.h
@@ -105,6 +105,11 @@ class CORE_EXPORT QgsApplication : public QApplication
     */
     static const QString licenceFilePath();
 
+	/*!
+	  Returns the path to the Recod doc files.
+	*/
+	static const QString recodAboutFilesPath();
+
     //! Returns the path to the help application.
     static const QString helpAppPath();
 
diff --git a/src/ui/qgsabout.ui b/src/ui/qgsabout.ui
index b11501c..495eb70 100644
--- a/src/ui/qgsabout.ui
+++ b/src/ui/qgsabout.ui
@@ -82,37 +82,17 @@
          </property>
          <item>
           <property name="text">
-           <string>About</string>
+           <string>RIAS RKD</string>
           </property>
          </item>
          <item>
           <property name="text">
-           <string>What's New</string>
+           <string>TMSforRES</string>
           </property>
          </item>
          <item>
           <property name="text">
-           <string>Providers</string>
-          </property>
-         </item>
-         <item>
-          <property name="text">
-           <string>Developers</string>
-          </property>
-         </item>
-         <item>
-          <property name="text">
-           <string>Contributors</string>
-          </property>
-         </item>
-         <item>
-          <property name="text">
-           <string>Translators</string>
-          </property>
-         </item>
-         <item>
-          <property name="text">
-           <string>Donors</string>
+           <string>Core description</string>
           </property>
          </item>
          <item>
@@ -150,8 +130,28 @@
           </sizepolicy>
          </property>
          <property name="currentIndex">
-          <number>0</number>
+          <number>9</number>
          </property>
+         <widget class="QWidget" name="page_RAIS_RKD_About">
+          <layout class="QVBoxLayout" name="verticalLayout_5">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QTextBrowser" name="RAIS_RKD_About_tb"/>
+           </item>
+          </layout>
+         </widget>
+         <widget class="QWidget" name="page_TMS_About">
+          <layout class="QVBoxLayout" name="verticalLayout_6">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QTextBrowser" name="TMS_About_tb"/>
+           </item>
+          </layout>
+         </widget>
          <widget class="QWidget" name="mOptsPage_01">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="margin">
@@ -191,8 +191,8 @@ p, li { white-space: pre-wrap; }
             </layout>
            </item>
            <item>
-            <widget class="QWebView" name="txtVersion" native="true">
-             <property name="url" stdset="0">
+            <widget class="QWebView" name="txtVersion">
+             <property name="url">
               <url>
                <string>about:blank</string>
               </url>
@@ -271,6 +271,16 @@ p, li { white-space: pre-wrap; }
            </item>
           </layout>
          </widget>
+         <widget class="QWidget" name="page_License">
+          <layout class="QVBoxLayout" name="verticalLayout_11">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QTextBrowser" name="txtLicense"/>
+           </item>
+          </layout>
+         </widget>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="margin">
@@ -365,8 +375,8 @@ p, li { white-space: pre-wrap; }
             <number>0</number>
            </property>
            <item>
-            <widget class="QWebView" name="txtTranslators" native="true">
-             <property name="url" stdset="0">
+            <widget class="QWebView" name="txtTranslators">
+             <property name="url">
               <url>
                <string>about:blank</string>
               </url>
@@ -392,16 +402,6 @@ p, li { white-space: pre-wrap; }
            </item>
           </layout>
          </widget>
-         <widget class="QWidget" name="page_8">
-          <layout class="QVBoxLayout" name="verticalLayout_11">
-           <property name="margin">
-            <number>0</number>
-           </property>
-           <item>
-            <widget class="QTextBrowser" name="txtLicense"/>
-           </item>
-          </layout>
-         </widget>
         </widget>
        </item>
       </layout>
