diff --git a/i18n/qgis_ru.ts b/i18n/qgis_ru.ts
index efe34d6..906f5d5 100644
--- a/i18n/qgis_ru.ts
+++ b/i18n/qgis_ru.ts
@@ -76651,4 +76651,30 @@ Convert a string into Qt data type.
         <translation>...</translation>
     </message>
 </context>
+<context>
+    <name>QgisApp</name>
+    <message>
+        <source>Target monitoring system of road economy</source>
+        <translation>Целевая система мониторинга дорожного хозяйства</translation>
+    </message>
+</context>
+<context>
+    <name>QgsAbout</name>
+    <message>
+        <source>RIAS RKD</source>
+        <translation>РИАС РКД</translation>
+    </message>
+    <message>
+        <source>TMSforDH</source>
+        <translation>ЦСМ ДХ</translation>
+    </message>
+    <message>
+        <source>Core description</source>
+        <translation>Описание ядра</translation>
+    </message>
+    <message>
+        <source>Developer</source>
+        <translation>Разработчик</translation>
+    </message>
+</context>
 </TS>
diff --git a/src/app/main.cpp b/src/app/main.cpp
index 3220506..4514693 100644
--- a/src/app/main.cpp
+++ b/src/app/main.cpp
@@ -478,6 +478,7 @@ int main( int argc, char *argv[] )
   QString pythonfile;
 
   QString customizationfile;
+  QString myWorkspace;
 
 #if defined(ANDROID)
   QgsDebugMsg( QString( "Android: All params stripped" ) );// Param %1" ).arg( argv[0] ) );
@@ -560,6 +561,10 @@ int main( int argc, char *argv[] )
       {
         myRestoreDefaultWindowState = true;
       }
+	  else if ( arg == "--workspace")
+      {
+        myWorkspace = QDir::toNativeSeparators( QDir( args[++i] ).absolutePath() );;
+      }
       else
       {
         myFileList.append( QDir::toNativeSeparators( QFileInfo( args[i] ).absoluteFilePath() ) );
@@ -688,6 +693,8 @@ int main( int argc, char *argv[] )
     }
   }
 
+  // set workspace
+  mySettings.setValue("/UI/lastProjectDir", myWorkspace);
 
   // custom environment variables
   QMap<QString, QString> systemEnvVars = QgsApplication::systemEnvVars();
diff --git a/src/app/qgisapp.cpp b/src/app/qgisapp.cpp
index defcf10..3e07f47 100644
--- a/src/app/qgisapp.cpp
+++ b/src/app/qgisapp.cpp
@@ -340,6 +340,7 @@ const int AFTER_RECENT_PATHS = 321;
   */
 static void setTitleBarText_( QWidget & qgisApp )
 {
+  /*
   QString caption = QgisApp::tr( "QGIS " );
 
   if ( QString( QGis::QGIS_VERSION ).endsWith( "Master" ) )
@@ -368,7 +369,8 @@ static void setTitleBarText_( QWidget & qgisApp )
   {
     caption += " - " + QgsProject::instance()->title();
   }
-
+  */
+  QString caption = QgisApp::tr( "Target monitoring system of road economy" );
   qgisApp.setWindowTitle( caption );
 } // setTitleBarText_( QWidget * qgisApp )
 
@@ -703,7 +705,10 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, QWidget * parent,
   connect( QgsMapLayerActionRegistry::instance(), SIGNAL( changed() ), this, SLOT( refreshActionFeatureAction() ) );
 
   // set application's caption
+  /*
   QString caption = tr( "QGIS - %1 ('%2')" ).arg( QGis::QGIS_VERSION ).arg( QGis::QGIS_RELEASE_NAME );
+  */
+  QString caption = QgisApp::tr( "Target monitoring system of road economy" );
   setWindowTitle( caption );
 
   QgsMessageLog::logMessage( tr( "QGIS starting..." ), QString::null, QgsMessageLog::INFO );
diff --git a/src/app/qgsabout.cpp b/src/app/qgsabout.cpp
index 1ff924a..5a3e0ab 100644
--- a/src/app/qgsabout.cpp
+++ b/src/app/qgsabout.cpp
@@ -38,6 +38,79 @@ QgsAbout::QgsAbout( QWidget *parent )
 #endif
 {
   setupUi( this );
+
+  QSettings qgisSettings;
+
+  TextLabel4->setText(TextLabel4->text() + " (build for Rekod, version 0.0.10) ");
+
+  // RIAS RKD about -------------------------
+  const QString RIAS_RKD_About_FileName(QgsApplication::recodAboutFilesPath() + "/RIAS_RKD_About");
+
+  QFile RIASRKDFile( RIAS_RKD_About_FileName );
+
+  QString riasrkdHTML = "";
+  if ( RIASRKDFile.open( QIODevice::ReadOnly ) )
+  {
+	
+    QTextStream translatorStream( &RIASRKDFile );
+    translatorStream.setCodec( "UTF-8" );
+    while ( !translatorStream.atEnd() )
+    {
+      riasrkdHTML += translatorStream.readLine();
+    }
+  }
+  else
+  {
+	riasrkdHTML += "<h1>" + RIAS_RKD_About_FileName + " not found </h1>";
+  }
+  RIAS_RKD_About_tb->setHtml( riasrkdHTML );
+  // -----------------------------------
+
+  // TMS about -------------------------  
+  const QString TMS_About_FileName(QgsApplication::recodAboutFilesPath() + "/TMSforDH_About");
+  
+  QFile TMSFile( TMS_About_FileName );
+
+  QString TMSHTML = "";
+  if ( TMSFile.open( QIODevice::ReadOnly ) )
+  {
+	
+    QTextStream translatorStream( &TMSFile );
+    translatorStream.setCodec( "UTF-8" );
+    while ( !translatorStream.atEnd() )
+    {
+      TMSHTML += translatorStream.readLine();
+    }
+  }
+  else
+  {
+	TMSHTML += "<h1>" + TMS_About_FileName + " not found </h1>";
+  }
+  TMS_About_tb->setHtml( TMSHTML );
+  // -----------------------------------
+
+  // Developer about ------------------------- 
+  const QString Developer_About_FileName(QgsApplication::recodAboutFilesPath() + "/Developer_About");
+  QFile DeveloperFile( Developer_About_FileName );
+  QString DeveloperHTML = "";
+  if ( DeveloperFile.open( QIODevice::ReadOnly ) )
+  {
+	
+    QTextStream translatorStream( &DeveloperFile );
+    translatorStream.setCodec( "UTF-8" );
+    while ( !translatorStream.atEnd() )
+    {
+      DeveloperHTML += translatorStream.readLine();
+    }
+  }
+  else
+  {
+	DeveloperHTML += "<h1>" + Developer_About_FileName + " not found </h1>";
+  }
+  Developer_About_tb->setHtml( DeveloperHTML );
+  
+  // -----------------------------------
+  
   QString title = QString( "%1 - %2 Bit" ).arg( windowTitle() ).arg( QSysInfo::WordSize );
   initOptionsBase( true, title );
   init();
diff --git a/src/app/qgscustomization.cpp b/src/app/qgscustomization.cpp
index 460d8e4..f25840d 100644
--- a/src/app/qgscustomization.cpp
+++ b/src/app/qgscustomization.cpp
@@ -652,6 +652,25 @@ void QgsCustomization::updateMainWindow( QMenu * theToolBarMenu )
   QMainWindow* mw = QgisApp::instance();
   QMenuBar* menubar = mw->menuBar();
 
+  // hide Panels
+  mSettings->beginGroup( "Customization/Panels" );
+  foreach ( QObject* obj, mw->children() )
+  {
+    if ( obj->inherits( "QDockWidget" ) )
+    {
+	  QDockWidget* bdw = qobject_cast<QDockWidget*>( obj );
+	  bool visible = mSettings->value( bdw->objectName(), true ).toBool();
+	
+      if ( !visible )
+      {
+		  mw->removeDockWidget(bdw);
+		  QgisApp::instance()->panelMenu()->removeAction( bdw->toggleViewAction() );
+      }
+    }
+  }
+
+  mSettings->endGroup();
+
   mSettings->beginGroup( "Customization/Menus" );
 
   // hide menus and menu actions
diff --git a/src/core/qgsapplication.cpp b/src/core/qgsapplication.cpp
index 1517824..a8c033e 100644
--- a/src/core/qgsapplication.cpp
+++ b/src/core/qgsapplication.cpp
@@ -458,6 +458,14 @@ const QString QgsApplication::licenceFilePath()
 }
 
 /*!
+  Returns the path to the Recod doc files.
+*/
+const QString QgsApplication::recodAboutFilesPath()
+{
+  return ABISYM( mPkgDataPath ) + QString( "/doc/Rekod" );
+}
+
+/*!
   Returns the path to the help application.
 */
 const QString QgsApplication::helpAppPath()
diff --git a/src/core/qgsapplication.h b/src/core/qgsapplication.h
index c4b4676..6fe8537 100644
--- a/src/core/qgsapplication.h
+++ b/src/core/qgsapplication.h
@@ -105,6 +105,11 @@ class CORE_EXPORT QgsApplication : public QApplication
     */
     static const QString licenceFilePath();
 
+	/*!
+	  Returns the path to the Recod doc files.
+	*/
+	static const QString recodAboutFilesPath();
+
     //! Returns the path to the help application.
     static const QString helpAppPath();
 
diff --git a/images/images.qrc b/images/images.qrc
index 39e3172..b3a2a13 100644
--- a/images/images.qrc
+++ b/images/images.qrc
@@ -473,4 +473,13 @@
         <file>sponsors/platinum.png</file>
         <file>sponsors/silver.png</file>
     </qresource>
+    <qresource prefix="/images/rekod">
+        <file>rekod/logo.png</file>
+    </qresource>
+    <qresource prefix="/images/nextgis">
+        <file>nextgis/logo.png</file>
+    </qresource>
+    <qresource prefix="/images/rekod_tms_4_doc">
+        <file alias="TMSforDH.jpg">rekod_tms_4_doc/TMSforDH.jpg</file>
+    </qresource>
 </RCC>
diff --git a/src/ui/qgsabout.ui b/src/ui/qgsabout.ui
index b11501c..33d96fd 100644
--- a/src/ui/qgsabout.ui
+++ b/src/ui/qgsabout.ui
@@ -82,42 +82,27 @@
          </property>
          <item>
           <property name="text">
-           <string>About</string>
+           <string>RIAS RKD</string>
           </property>
          </item>
          <item>
           <property name="text">
-           <string>What's New</string>
+           <string>TMSforDH</string>
           </property>
          </item>
          <item>
           <property name="text">
-           <string>Providers</string>
+           <string>Core description</string>
           </property>
          </item>
          <item>
           <property name="text">
-           <string>Developers</string>
-          </property>
-         </item>
-         <item>
-          <property name="text">
-           <string>Contributors</string>
-          </property>
-         </item>
-         <item>
-          <property name="text">
-           <string>Translators</string>
-          </property>
-         </item>
-         <item>
-          <property name="text">
-           <string>Donors</string>
+           <string>License</string>
           </property>
          </item>
          <item>
           <property name="text">
-           <string>License</string>
+           <string>Developer</string>
           </property>
          </item>
         </widget>
@@ -150,8 +135,32 @@
           </sizepolicy>
          </property>
          <property name="currentIndex">
-          <number>0</number>
+          <number>1</number>
          </property>
+         <widget class="QWidget" name="page_RIAS_RKD_About">
+          <layout class="QVBoxLayout" name="verticalLayout_5">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QTextBrowser" name="RIAS_RKD_About_tb"/>
+           </item>
+          </layout>
+         </widget>
+         <widget class="QWidget" name="page_TMS_About">
+          <layout class="QVBoxLayout" name="verticalLayout_6">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QTextBrowser" name="TMS_About_tb">
+             <property name="styleSheet">
+              <string notr="true">width:400</string>
+             </property>
+            </widget>
+           </item>
+          </layout>
+         </widget>
          <widget class="QWidget" name="mOptsPage_01">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="margin">
@@ -191,8 +200,8 @@ p, li { white-space: pre-wrap; }
             </layout>
            </item>
            <item>
-            <widget class="QWebView" name="txtVersion" native="true">
-             <property name="url" stdset="0">
+            <widget class="QWebView" name="txtVersion">
+             <property name="url">
               <url>
                <string>about:blank</string>
               </url>
@@ -271,6 +280,26 @@ p, li { white-space: pre-wrap; }
            </item>
           </layout>
          </widget>
+         <widget class="QWidget" name="page_License">
+          <layout class="QVBoxLayout" name="verticalLayout_11">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QTextBrowser" name="txtLicense"/>
+           </item>
+          </layout>
+         </widget>
+         <widget class="QWidget" name="page_Developer_About">
+          <layout class="QVBoxLayout" name="verticalLayout_51">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QTextBrowser" name="Developer_About_tb"/>
+           </item>
+          </layout>
+         </widget>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="margin">
@@ -365,8 +394,8 @@ p, li { white-space: pre-wrap; }
             <number>0</number>
            </property>
            <item>
-            <widget class="QWebView" name="txtTranslators" native="true">
-             <property name="url" stdset="0">
+            <widget class="QWebView" name="txtTranslators">
+             <property name="url">
               <url>
                <string>about:blank</string>
               </url>
@@ -392,16 +421,6 @@ p, li { white-space: pre-wrap; }
            </item>
           </layout>
          </widget>
-         <widget class="QWidget" name="page_8">
-          <layout class="QVBoxLayout" name="verticalLayout_11">
-           <property name="margin">
-            <number>0</number>
-           </property>
-           <item>
-            <widget class="QTextBrowser" name="txtLicense"/>
-           </item>
-          </layout>
-         </widget>
         </widget>
        </item>
       </layout>
